docs/examples/extensions/add-task/woocommerce-admin-add-task-example.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Plugin Name: WooCommerce Admin Add Task Example
+ *
+ * @package WC_Admin
+ */
+
+/**
+ * Register the JS.
+ */
+function add_task_register_script() {
+
+	if ( ! class_exists( 'Automattic\WooCommerce\Admin\Loader' ) || ! \Automattic\WooCommerce\Admin\Loader::is_admin_page() ) {
```

Comment:
It would be good for the example to also do a `should_show_tasks()` check. That way, if people use these examples, their code is only loaded when necessary.